program/src/instructions/compressed/buy_spl.rs

```diff
@@ -193,12 +212,14 @@ pub fn process_buy_spl<'info>(
             ctx.remaining_accounts
         };
 
+    // TODO: should we really have this in the oss contract?
     // NB: TRoll hardcodes Some(100) to match
```

Comment:
This just means we enforce royalties on cNFTs, like good NFT ecosystem citizens.